post-content.js

```diff
@@ -6,163 +6,172 @@ window._wpGutenbergPost = {
-
-			'<!-- wp:core/quote style="2" -->',
-			'<blockquote class="blocks-quote-style-2"><p>There is no greater agony than bearing an untold story inside you.</p><footer>Maya Angelou</footer></blockquote>',
-			'<!-- /wp:core/quote -->',
-
-			'<!-- wp:core/separator -->',
-			'<hr/>',
-			'<!-- /wp:core/separator -->',
-
-			'<!-- wp:core/heading -->',
-			'<h2>Sea id autem nominavi deseruisse</h2>',
-			'<!-- /wp:core/heading -->',
-
-			'<!-- wp:core/text -->',
-			'<p>Ea veniam homero eam. Ex inimicus molestiae cum, debet scaevola at eos. Vis assum veritus ut, has ea nostrud accusata, offendit appareat comprehensam ea pro. Ad quo quem veritus appellantur, te est quas phaedrum, eum alia habeo ad. Ei est erroribus imperdiet, omnis dicam propriae sed no. His vitae oratio fierent ne, cu duo tota eligendi, electram rationibus in qui.</p>',
-			'<!-- /wp:core/text -->',
-
-			'<!-- wp:core/image -->',
-			'<figure><img src="https://cldup.com/uuUqE_dXzy.jpg" /></figure>',
-			'<!-- /wp:core/image -->',
-
-			'<!-- wp:core/text -->',
-			'<p>Est quis reque cetero ad. Sea id autem nominavi deseruisse. <strong>Veniam qualisque definitionem pri id</strong>, ea autem feugiat delenit ius, mei at loem affert accumsan. Dicat eruditi cu est, te pro dicant pericula conclusionemque, ei vim detracto euripidis intellegam. Eius postea volumus mei ad.</p>',
-			'<p>Prima ridens denique his te, ferri illum volumus an his. Eu vel dicat homero qualisqu, vitae regione deserunt vis ei. Graeci incorrupte liberavisse no mea, saepe voluptaria usu ex, vis dicant euismod id. At dolor reprimique eos, quo altera detraxit moderatius id. Quo iudico utinam eu, ad alia munere mel.</p>',
-			'<!-- /wp:core/text -->',
-
-			'<!-- wp:core/code -->',
-			'<pre><code>export default function MyButton() {\n\
-	return &lt;Button&gt;Click Me!&lt;/Button&gt;;\n\
-}</code></pre>',
-			'<!-- /wp:core/code -->',
+		raw:
+`
+<!-- wp:core/text -->
+<p>The goal of this new editor is to make adding rich content to WordPress simple and enjoyable. This whole post is composed of <em>pieces of content</em>—somewhat similar to LEGO bricks—that you can move around and interact with. Move your cursor around and you\'ll notice the different blocks light up with outlines and arrows. Press the arrows to reposition blocks quickly, without fearing about losing things in the process of copying and pasting.</p>
+<p>What you are reading now is a <strong>text block</strong>, the most basic block of all. A text block can have multiple paragraphs, if that\'s how you prefer to write your posts. But you can also split it by hitting enter twice. Once blocks are split they get their own controls to be moved freely around the post...</p>
+<!-- /wp:core/text -->
 
-			'<!-- wp:core/image -->',
-			'<figure><img src="https://cldup.com/GCwahb3aOb.jpg" /></figure>',
-			'<!-- /wp:core/image -->',
+<p class="fancy important">This is just a plain <em>old</em> block in a <pre><code>&lt;p&gt;</code></pre> tag.</p>
```

Comment:
That's an interesting addition, because I'm not sure if we should accept a text block with a code _block_ in it. That's a job for validation I guess. cc @nylen.